release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2892,6 +2892,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.29
+[id="ocp-4-17-29_{context}"]
+=== RHSA-2025:4723 - {product-title} {product-version}.29 bug fix and security update advisory
+
+Issued: 15 May 2025
+
+{product-title} release {product-version}.29 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4723[RHSA-2025:4723] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4433[RHBA-2025:4725] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.29 --pullspecs
+----
+
+[id="ocp-4-17-29-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a race condition in the hostname that handled the code caused inconsistencies between the boot and machine hostnames. With this release, the race condition is resolved, which ensures consistent hostnames in the Ignition configuration file during the operating system installation. (link:https://issues.redhat.com/browse/OCPBUGS-55680[OCPBUGS-55680]). 
```

Comment:
This text doesn't seem to match the bug linked. It's not really clear when this happens (is "OS installation" referring to some operating system, or OpenShift?) or for what environments - seems it has something to do with the Bare Metal Operator but that isn't mentioned in the writeup :thinking: 

Maybe more like this?
```suggestion
* Previously, a race condition caused inconsistencies between the node and machine hostnames. With this release, the race condition is resolved, which ensures consistent hostnames in the Ignition configuration file during installation. (link:https://issues.redhat.com/browse/OCPBUGS-55680[OCPBUGS-55680]). 
```

That's just based on my read of the bug, should get an SME to check this.